core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/signing/SigningValidator.java

```diff
@@ -69,13 +69,19 @@ public static void validateSigning(JReleaserContext context, Mode mode, Errors e
 
         boolean cosign = signing.resolveMode() == org.jreleaser.model.Signing.Mode.COSIGN;
 
-        signing.setPassphrase(
-            checkProperty(context,
-                cosign ? COSIGN_PASSWORD : GPG_PASSPHRASE,
-                "signing.passphrase",
-                signing.getPassphrase(),
-                errors,
-                context.isDryrun()));
+        String passphrase = checkProperty(context,
+            cosign ? COSIGN_PASSWORD : GPG_PASSPHRASE,
+            "signing.passphrase",
+            signing.getPassphrase(),
+            new Errors(),
```

Comment:
I tried to minimise my changes. I pass `new Errors()` here as a way to ignore the errors (otherwise it just fails).

Reply:
Ok, right. Then please add a message resource for the new warning message stating that the passphrase is blank. And if one should be supplies then use the suggested env vars or sys props.